### DISCUSSION TITLE
Cross platform support!

### DISCUSSION THREAD

**Discussion by Azyyyyyy** [FluxpointDev/MultiRPC#21] [Old]
MultiRPC now has a stable(ish) cross platform build which contains the main functionally! (No macOS .app yet but you can build the project yourself, just note there might be issues which we don't know about!)

<h3>What it's still missing:</h3>

- [x] Settings + UI
- [ ] Builds setup for being updatable (The logic for updates is in, just need to setup for making update files)
- [ ] Update UI
- [x] Theming page
- [x] Credit page
- [x] Debug page
- [x] Network notification
- [x] Show/Hide Tooltray
- [x] Administrator check
- [x] One process check
- [x] Discord check
- [x] Showing current version
- [x] Ability to have custom side bar icon's (Not just default colours which is not used with Custom's page icon)

<h3>To readd later</h3>

- [ ]  JumpLists (I think this never actually worked in stable anyway)
- [ ] Taskbar thumbnail

Also should add https://github.com/FluxpointDev/MultiRPC/discussions/16#discussion-3419796 into this version

Once this has been made the stable we need to work on https://github.com/FluxpointDev/MultiRPC/discussions/19 and https://github.com/FluxpointDev/MultiRPC/discussions/15

<hr /><em>This discussion was created from the release <a href='https://github.com/FluxpointDev/MultiRPC/releases/tag/'>v7.0-crossplat</a>.</em>

**Comment by xXBuilderBXx** [comment]
Redoing MultiRPC
